.gitmodules

```diff
@@ -4,3 +4,9 @@
 [submodule "externals/REFPROP-cmake"]
 	path = externals/REFPROP-cmake
 	url = https://github.com/usnistgov/REFPROP-cmake
+[submodule "wrappers/Mathcad/externals/REFPROP-headers"]
```

Comment:
Probably, but:
1. I'm preparing to add in the REFPROP, REFPROP1, and REFPROP2 calls from version 10 and the one in top level externals is not updated.
2. I rather like having everything "under" the Mathcad directory, in case users want just that directory.

If that's the route you'd rather go, I can certainly reconfigure the build solutions and bump the one in REFPROP-wrappers/externals.  I'll need to add fmt(fmtlib) up there as well.

Reply:
done.